src/app/static/src/app/components/UserHeader.vue

```diff
@@ -113,7 +113,7 @@
         computed: {
             ...mapStateProps<LoadState, keyof LoadComputed>("load", {
                 hasError: state => state.loadingState==LoadingState.LoadFailed,
-                loadError: state => state.loadError
+                loadError: state => state.loadError && state.loadError.detail
```

Comment:
One of the other ways this error gets populated is if the checksum doesn't match, in which case looks like a string gets committed as the `loadError` at the moment so won't have a `detail` property, should be changed to an error object: https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/store/load/actions.ts#L34